docs/app/pages/getting-started/getting-started.component.html

```diff
@@ -1,6 +1,21 @@
 <uxd-page-header header="Getting Started"></uxd-page-header>
 <uxd-text-page-layout>
-    <h3>Installation</h3>
+
+    <h3 id="add-to-an-existing-project">Add to an Existing Project</h3>
+
+    <p>
+        To add UX Aspects to your Angular CLI or Nx workspace, run the following command in the project root:
```

Comment:
I would prefer to always refer to the library as "UX Aspects (AngularJS)" here, to minimize any potential confusion about what these instructions are referring to.